Xamarin.Forms.Xaml/MarkupExtensions/OnIdiomExtension.cs

```diff
@@ -22,20 +22,32 @@ public class OnIdiomExtension : IMarkupExtension
 
 		public object ProvideValue(IServiceProvider serviceProvider)
 		{
-			var lineInfo = serviceProvider?.GetService<IXmlLineInfoProvider>()?.XmlLineInfo;
-			if (Default == null && Phone == null &&
-				Tablet == null && Desktop == null && TV == null && Watch == null)
-			{
+			if (   Default == null
```

Comment:
This code path not covered by test.

Reply:
an equivalent codepath (OnPlatform) is tested

Reply:
Hehe, cute. Duplicating code and covering one path and not the other is honky-dory, huh? I'm not gonna loose any sleep over this code not being covered -- it just seems an opportune moment to enhance the stubbing framework...